app/views.py

```diff
@@ -1,12 +1,26 @@
 from django.db.models import Q
 from django.shortcuts import render
+from django.http import JsonResponse
+from django.core.serializers.json import DjangoJSONEncoder
 
-from app.models import Player, Rank
+from app.models import Player, Rank, Race
 
-def index(request):
-    return render(request, 'index.html')
+class EnumEncoder(DjangoJSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, (Rank, Race)):
+            return str(obj)
+        else:
+            return super().default(obj)
 
-def search(request):
+
+def api_search(request):
+    players = list(
+        get_players(request).values('realm', 'region', 'rank', 'username', 'bnet_id', 'race',
+                                    'mmr', 'wins', 'losses', 'clan', 'profile_id'))
+    return JsonResponse(players, safe=False, encoder=EnumEncoder)
+
+
+def get_players(request):
```

Comment:
That's cute using request here. Not sure if this is the best idea but I'm down with it